spec/rspec/core/bisect/runner_spec.rb

```diff
@@ -223,6 +224,7 @@ def repro_command_from(ids)
       end
 
       it "runs the suite with the locations from the original CLI args" do
+        pending "this is not how it works anymore"
         runner.original_results
         expect(Open3).to have_received(open3_method).with(a_string_including("spec/unit"))
```

Comment:
I think this spec can be adjusted to the new way things are working by changing it to something like:

``` ruby
let(:original_cli_args) { %w[spec/unit --seed 1234] }

it "runs the suite with the original CLI args" do
  runner.original_results
  expect(Open3).to have_received(open3_method).with(a_string_including("--seed 1234"))
end
```

The point of this spec is showing that it runs the suite, passing along CLI args appropriately.  The location was used before but we can just update the `original_cli_args` to include another arg like seed and set an expectation about that.